imports/components/controls.jsx

```diff
@@ -0,0 +1,204 @@
+import { DropdownButton, MenuItem, Modal, Button } from 'react-bootstrap';
+import { PlayerList } from './player_list';
+
+class RevealControl extends React.Component {
```

Comment:
Yeah, I'm still learning React and figuring out the prop structure, and it looks like `propTypes` is no longer core, so I ran out of yak shears before getting to it. I'll probably do a pass once I'm more confident I like the structure.